doc/cygnus-ngsi/installation_and_administration_guide/backends_as_sth.md

```diff
@@ -0,0 +1,47 @@
+#<a name="top"></a>Backends as short-term historics
+Backends are used by Cygnus NGSI as "infinite" historical context data repositories. More and more data is appended to files, tables and collections as data flow from a NGSI source. Such a data flow may never end, thus, insertions may never end too, exhausting the available storing resources.
+
+Therefore, it is important to provide mechanisms in charge of controlling how much data is stored in the persistence backends, removing old data in favour of new one, resulting in some kind of short-term historic implementation.
+
+From version 1.7.0 this is something that can be done by means of the **capping** and/or **expirating** features.
+
+##How it works
+There are two approaches when deciding which data must be removed from existent historics:
+
+* By **capping** data "records"(*) once certain size limit has been reached. In other words, to ensure that only the last N records are stored, honouring the capping limit in place.
+* By **expirating** "records"(*) once certain keepalive limit has been reached. In other words, to control only records added in the last N seconds are maintained, deleting old ones.
+
+The above can be controlled by means of specific configuration parameters in certain sinks:
+
+| Parameter | Mandatory | Default value | Comments |
+|---|---|---|---|
+| persistence\_policy.max_records | no | -1 | Maximum number of records allowed for a persistence element (table or resource) before it is capped. `-1` disables this policy. |
```

Comment:
table is a MySQL concept, resource a CKAN concetp... why don't to mention JSON file rows (HDFS concept) or collections (MongoDB concept)?

Thus, I'd suggest some of the following alternatives:

* Remove `(table or resource)` to make the sentences neutral
* Use it as an "open list", meaning non-exahustive examples, I  mean:  `(table, resource, etc.)`

I think the second one is better.